### PR TITLE
examples/pod_exec.py: fix example

### PR DESCRIPTION
**Pull request by Akasurde**
Use get_default_copy() API with configuration.

### CODE REVIEW

**Comment by Akasurde** [comment]
Fixes: #1396

File: examples/pod_exec.py [hunk 1]
```diff
@@ -117,8 +117,11 @@ def exec_commands(api_instance):
 
 def main():
     config.load_kube_config()
-    c = Configuration()
-    c.assert_hostname = False
+    try:
+        c = Configuration().get_default_copy()
```
Comment (roycaihw) [inline examples/pod_exec.py:121]: nit: maybe add a comment that this is the right way of setting the config starting v12.0.0

**Review by roycaihw — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**